circle.yml

```diff
@@ -17,8 +17,8 @@ notify:
 machine:
   # Overwrite these ENV variables in parametrized (manual/API) builds
   environment:
-    DISTROS: "wheezy jessie trusty el6 el7"
-    NOTESTS: "el7"
+    DISTROS: "wheezy jessie trusty xenial el6 el7"
+    NOTESTS: "xenial el7"
```

Comment:
Most likely the same reason as with xenial - you need to have a privileged container to run systemd and circleci doesn't allow you to do that. At least that's my assumption on what's going on, I don't really have deep enough understanding of either systemd or docker.